charts/publisher/values.yaml

```diff
@@ -1,121 +1,118 @@
 # Default values for Publisher.
 
-govukEnvironment: test
-govukDomainExternal: govuk.digital
-govukDomainInternal: govuk-internal.digital
-clusterDomain: svc.cluster.local
-
-service:
-  type: NodePort
-  annotations: {}
-  port: 80
-
-webReplicaCount: 1
-workerReplicaCount: 1
-
-# Publisher uses the same image version in both Web and Worker mode.
-appImage:
-  repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/publisher
-  pullPolicy: Always
-  tag: latest
-
-nginxImage:
-  repository: nginx
-  pullPolicy: Always
-  tag: stable
-
-nginxPort: 80
-appPort: 3000
-
-webResources: {}
+# Values common to both Web and Worker deployments
+common:
+  govukEnvironment: test
+  govukDomainExternal: govuk.digital
+  govukDomainInternal: svc.cluster.local
+
+  image: &commonImage
+    repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/publisher
+    pullPolicy: Always
+    tag: latest
+
+  secrets:
+    externalSecretRef:
+    externalSecretRefreshInterval: "5m"
```

Comment:
Just spotted this - I'd recommend bumping this up a bit to minimise load on etcd. (Secrets Manager can take a hammering but we'll easily have hundreds of secrets once we have Signon and all the apps running, and rotation will be on the order of ~monthly so we should be totally fine with say half an hour or even the default 1h. It might also make it easier to spot transient issues e.g. if a buggy rotation script is revoking previous credentials before all the clients have switched to new ones.)